jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -170,3 +182,16 @@
     </div>
   </j-tray-plugin>
 </template>
+
+<script>
+
+module.exports = {
+  methods: {
+    predictedMemoryUsagePerc(npoints) {
+      const slope = 8500
+      const memory = performance.memory
+      return slope * npoints / (memory.jsHeapSizeLimit - memory.totalJSHeapSize) * 100

```

Comment:
Maybe this is more useful? The percentage of *remaining* memory is a moving target. Or even better, get rid of `* 100`.
```suggestion
      return slope * npoints / memory.jsHeapSizeLimit * 100

```

Reply:
Also hard for me to determine whether this is accurate or not unless it also reports actual memory used in generating the plot.

Reply:
(moved to #1821)